provisioning/device/samples/ProvisioningDeviceClientX509/readme.md

```diff
@@ -0,0 +1,53 @@
+# Provisioning Device Client Sample - TPM Attestation
+
+## Overview
+
+This is a quick tutorial with the steps to register a device in the Microsoft Azure IoT Hub Device Provisioning Service using X509 certificates locally generated.
+
+## How to run the sample
+
+Ensure that all prerequisite steps presented in [samples](..\) have been performed.
+
+The sample code is set up to use X509 certificates stored within a password-protected PKCS12 formatted file (certificate.pfx). To generate a self-signed certificate run the following command:
```

Comment:
Nit again, missing the dot in X.509

Reply:
Will fix here and everywhere I can find X509 in docs.